kuma/wiki/models.py

```diff
@@ -1467,6 +1473,32 @@ def contributors(self):
     def zone_stack(self):
         return DocumentZoneStackJob().get(self.pk)
 
+    def set_share_url(self, save=True):
+        share_url = bitly_shorten('/'.join([settings.SITE_URL,
+                                            self.get_absolute_url()]))
+        if save and share_url:
+            self.share_url = share_url
+            if self.pk:
+                self.save(update_fields=['share_url'])
+            else:
+                self.save()
+
+    def get_share_url(self):
+        """
+        Returns the `share_url` or generates it if None.
```

Comment:
The best way I think as it would allow autodoc sections is this style: http://sphinx-doc.org/domains.html#info-field-lists